conf/deployment/rosa/ipi_3az_rhcos_3m_3w.yaml

```diff
@@ -0,0 +1,18 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'rosa'
+  deployment_type: 'ipi'
```

Comment:
Is this considered as IPI or rather `managed` ?

Reply:
It is managed. Thank You.